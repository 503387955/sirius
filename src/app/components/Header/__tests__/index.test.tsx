import React from 'react';
import { render } from '@testing-library/react';

import { Header } from '..';

describe('<Header  />', () => {
  it('should match snapshot', () => {
    const logo = <span key="1">header logo</span>;
    const left = [<span key="1">Blocks&Transaction</span>];
    const right = [<span key="1">EN</span>];
    const loadingIndicator = render(
      <Header logo={logo} left={left} right={right} />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
