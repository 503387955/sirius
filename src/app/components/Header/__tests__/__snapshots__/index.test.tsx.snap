// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header  /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<nav
  class="c0"
>
  <div
    class=""
  >
    <span>
      header logo
    </span>
  </div>
  <div
    class=""
  >
    <span>
      Blocks&Transaction
    </span>
  </div>
  <div
    class=""
  >
    <span>
      EN
    </span>
  </div>
</nav>
`;
